mrp_production_hierarchy/views/mrp_production.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+        <record id="mrp_production_form_view" model="ir.ui.view">
+            <field name="name">mrp.production.form</field>
+            <field name="model">mrp.production</field>
+            <field name="inherit_id" ref="mrp.mrp_production_form_view"/>
+            <field name="arch" type="xml">
+                <div name="button_box" position="inside">
+                    <field name="child_ids" invisible="1"/>
+                    <button name="open_production_tree" type="object"
+                        string="Hierarchy" class="oe_stat_button"
```

Comment:
As there is no hierarchical tree view anymore, we should to rename this label...

Reply:
There is no real hierarchical tree view, but an hand-made, as you can see:
![Screenshot(66)](https://user-images.githubusercontent.com/7657311/106312473-f9ad4300-6266-11eb-86eb-a862175a6095.png)